packages/sdk-snippets/src/helpers/utils.ts

```diff
@@ -0,0 +1,24 @@
+import type { ClientInfo, TargetId, TargetInfo } from '../targets/targets';
+import { targets } from '../targets/targets';
+
+export interface AvailableTarget extends TargetInfo {
+  clients: ClientInfo[];
+}
+
+export const availableWebhookTargets = () =>
+  Object.keys(targets).map<AvailableTarget>(targetId => ({
+    ...targets[targetId as TargetId].info,
+    clients: Object.keys(targets[targetId as TargetId].services.webhooks.clientsById).map(
+      clientId => targets[targetId as TargetId].services.webhooks.clientsById[clientId].info
+    ),
+  }));
+
+export const availableServerTargets = () =>
+  Object.keys(targets).map<AvailableTarget>(targetId => ({
+    ...targets[targetId as TargetId].info,
+    clients: Object.keys(targets[targetId as TargetId].services.server.clientsById).map(
+      clientId => targets[targetId as TargetId].services.server.clientsById[clientId].info
+    ),
+  }));
+
+export const extname = (targetId: TargetId) => targets[targetId]?.info.extname || '';
```

Comment:
May want to add deeper optional chaining as a worst case scenario.

Reply:
This'll get caught by TS